.github/workflows/generate-sandboxes-next.yml

```
@@ -21,7 +21,7 @@ jobs:
     steps:
       - uses: actions/checkout@v3
         with:
-          ref: next
+          ref: valentin/setup-sandboxes
```

reminder to bring this back to next
```suggestion
          ref: next
```